query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ServicesQueries.java

```diff
@@ -53,6 +55,66 @@ static QueryRequest buildQuery1() {
             .build());
 
 
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceId));
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceName));
+    Function serviceDurationPercentileFunc = Function.newBuilder().addArguments(
+        Expression.newBuilder().setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName("SERVICE.duration").build()).build()).setFunctionName("PERCENTILE99")
+        .build();
+    OrderByExpression orderByExpression = OrderByExpression.newBuilder().setOrder(SortOrder.DESC).setExpression(
+        Expression.newBuilder().setFunction(serviceDurationPercentileFunc).build()).build();
+    builder.addOrderBy(orderByExpression);
+    builder.setLimit(10000);
```

Comment:
Is this `10000` arbitrary? Is this declared or documented somewhere?

Reply:
Yes. arbitrary for sake of testing.